### PR TITLE
added quotes

### PR DESCRIPTION
**Pull request by christinaashworth**
I added quotes to the quote list section! (in index.html)

**How to check:** 
-fetch branch
-serve
-make sure quotes show up in DOM
-celebrate!!! yay!!!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,7 +59,14 @@ <h1>Tanks for the Memories</h1>
         </section>
       </article>
       <article class="quotesList">
-        all quotes
+        <section class="quote card">
+          <div class="quote__text">"Just Keep Swimming!"</div>
+          <div class="quote__author">--Dory</div>
+        </section>
+        <section class="quote card">
```
Comment (FORDBA) [inline index.html:66]: this looks great

File: index.html [hunk 2]
```diff
@@ -59,7 +59,14 @@ <h1>Tanks for the Memories</h1>
...
+          <div class="quote__text">"Fish are our friends"</div>
+          <div class="quote__author">--Bruce</div>
```
Comment (FORDBA) [inline index.html:68]: have you verified that Bruce said this?

**Review by FORDBA — COMMENTED** [review]
See my comments, but otherwise good to go

**Comment by FORDBA** [comment]
👍

File: index.html (hunk 1 above)
Comment (christinaashworth) [reply index.html:66]: thanks!

**Pull request merged**